### PR TITLE
Explorer transfer table1

### PR DESCRIPTION
**Pull request by Jibz1**
- Created Tabs and Table components that can be reused
- Update the home page to match the design using this component 
- Warning a lot of mock data in areas where we do not have it. We easily update or remove those components.
- update the timeUtil to include shorter displays

Screen Shot 2022-07-08 at 10 07 55 AM

https://user-images.githubusercontent.com/8676844/178000647-6e901401-0a40-4265-9f09-31023f565b45.mov

### CODE REVIEW

**Review by stella3d — CHANGES_REQUESTED** [review]
looks mostly good, i feel like this bundles together a few parts which should come separately.

We could really use this table view as a re-usable component for other kinds of data.  I would like it if we can break that out of the part of this PR that relies on mock data we don't have ready yet - would make it easier to review.

File: explorer/client/src/components/table/TableCard.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+type TableColumn = {
+    headerLabel: string;
+    accessorKey: string;
+};
+// TODO: update Link to use Tuple type
+// type Links = [Link, Link?];
+type Links = Link[];
+
+// support multiple types with sepcial handling for 'addresses'/links and status
```
[the remark this answers is not in the document]
Comment (Jibz1) [reply explorer/client/src/components/table/TableCard.tsx:35]: > sepcial -> special

Noted. 

Currently, the table view supports almost all input types. The `Links` and `Status` are just special cases with design support. 
Screen Shot 2022-07-08 at 11 15 39 AM

File: explorer/client/src/components/table/TableCard.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        | boolean
+        | Links
+        | React.ReactElement
+        | {
+              txTypeName: TransactionKindName | undefined;
+              status: ExecutionStatusType;
+          };
+};
+
+function TableCard({
```
[the remark this answers is not in the document]
Comment (Jibz1) [reply explorer/client/src/components/table/TableCard.tsx:118]: @stella3d this has been refactored. Please review.

**Comment by Jibz1** [comment]
Screen Shot 2022-07-08 at 3 08 29 PM @666lcz

File: explorer/client/src/components/transaction-card/RecentTxCard.tsx [hunk 3]
```diff
@@ -100,94 +116,96 @@ async function getRecentTransactions(
...
+            },
+        ],
+    };
+    const tabsFooter = {
+        link: {
+            text: '',
+            categoryName: 'transactions' as Category,
+            isLink: true,
+            isCopyButton: false,
+            alttext: 'More Transaction',
```
Comment (666lcz) [inline explorer/client/src/components/transaction-card/RecentTxCard.tsx:191]:
```suggestion
            alttext: 'More Transactions',
```

**Review by 666lcz — APPROVED** [review]

File: explorer/client/src/components/table/TableCard.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jibz1) [reply explorer/client/src/components/table/TableCard.tsx]: @stella3d this has been updated please review. Thanks

**Review by stella3d — APPROVED** [review]

**Pull request merged**